users/models.py

```diff
@@ -94,3 +141,19 @@ def last_activity_ago(self: "User") -> str:
             result = f"{delta.days} days ago"
 
         return result
+    
+    @property
+    def nickname(self):
```

Comment:
According to reddit, nickname is required so this "not yet named" is not needed.

Reply:
> That's all that should be here, I believe.

What should be Profile.nickname be on creation?  You're saying that it should always have value. It will be right after saving form. But before I have to create User. How do I avoid default or "" or None in signal (for training I'm taking signal rather than manager approach [for skills])

Reply:
Understood, this approach is good for learning
1. In this property, just leave "profile not created" or sth. It may happen especially for admin users or old users. 
2. I don't think we need to create user profle on user creation. We should create profile only when user is registered via user registration form, no? This way we will also not create "empty" objects for admin users.

Reply:
> Understood, this approach is good for learning
> 
>     1. In this property, just leave "profile not created" or sth. It may happen especially for admin users or old users.
> 
>     2. I don't think we need to create user profle on user creation. We should create profile only when user is registered via user registration form, no? This way we will also not create "empty" objects for admin users.

but manual user creation (lest's say with User.objects.create()) won't be functional

Reply:
aa ok I can change/add  UserManager.. Also is this ok that we have both `.create()` and `.create_user()` in UserManager?